packages/bundler/src/modules/BundleManagerRIP7560.ts

```diff
@@ -71,14 +74,42 @@ export class BundleManagerRIP7560 extends BundleManager implements IBundleManage
     }
   }
 
+  async createBundle (
+    minBaseFee: BigNumberish,
+    maxBundleGas: BigNumberish,
+    maxBundleSize: BigNumberish
+  ): Promise<[OperationBase[], StorageMap]> {
+    const [bundle, storageMap] = await super.createBundle(minBaseFee, maxBundleGas, maxBundleSize)
+    if (bundle.length === 0) {
+      return [[], {}]
+    }
+    const bundleHash = this.computeBundleHash(bundle)
+
+    // TODO: deduplicate this code with the PUSH method
+    const userOpHashes: string[] = []
+    for (const transaction of bundle) {
+      userOpHashes.push(getRIP7560TransactionHash(transaction as OperationRIP7560))
+    }
+    this.sentBundles.push({
+      transactionHash: bundleHash,
+      userOpHashes
+    })
+
+    return [bundle, storageMap]
+  }
+
   async sendBundle (userOps: OperationBase[], _beneficiary: string, _storageMap: StorageMap): Promise<any> {
+    if (this.useRip7560Mode === 'PULL') {
```

Comment:
why does it silently return ? either do something, or (more likely) return an error, since sendBundle is not allowed in pull mode.